recipes/ijson/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "ijson" %}
```

Comment:
This recipe looks fine to me. If you would like to break it out in a separate PR, I'd be happy to merge it.